D3D12Bundles/D3D12Bundles.cs

```diff
@@ -699,43 +651,41 @@ struct Vertex {
             // However, when ExecuteCommandList() is called on a particular command
             // list, that command list can then be reset at any time and must be before
             // re-recording.
-            mCommandList.Reset(mCurrentFrameResource.mCommandAllocator, mPipelineState1);
+            commandList.Reset(mCurrentFrameResource.mCommandAllocator, mPipelineState1);
 
             // Set necessary state.
-            mCommandList.SetGraphicsRootSignature(mRootSignature);
-
-            mCommandList.SetDescriptorHeaps(new[] { mCbvSrvHeap, mSamplerHeap });
```

Comment:
Here's another such call, using the `SetNecessaryState` to embody these two calls.